sdk/security/azure-security-nspkg/azure/__init__.py

```diff
@@ -0,0 +1 @@
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
Is this here to make editable installs work (for python 2)? It is not included in the actual package (wheel/sdist), correct?

Reply:
It's not in the wheel, but it's in the sdist. This applies to all packages (nspkg or not), precisely to simplify dev indeed.

Reply:
And I actually opened both wheel and sdist before doing the PR to check that was the case :p